v19.2/cockroachcloud-create-your-cluster.md

```diff
@@ -23,13 +23,10 @@ On the **Create new cluster** page, select either **Google Cloud** or **AWS** as
 
 The choice of the cloud provider decides the price per node. For pricing comparison, refer to the following table:
 
-Hardware configuration	| GCP Pricing (per node, per month)	| AWS Pricing (per node, per month)
+Hardware configuration	| GCP Pricing (per node, per hour)	| AWS Pricing (per node, per hour)
 ----------|------------|------------
-Small (2 vCPU, 60 GB disk) |	$75	| $180
-Medium	(4 vCPU, 100 GB disk) | $300 | $400
-Large (4 vCPU, 250 GB disk) | $540	| $700
-
-To enable VPC peering in the future, choose the cloud platform on which your application is deployed.
+Small (2 vCPU, 60 GB disk) |	$0.11	| $0.48
+Large (4 vCPU, 150 GB disk) | $0.69	| $0.83
```

Comment:
According to the spreadsheet we're using the names `Small` and `Medium`?

Reply:
Hmm. I referred to the screenshot in the PR: https://github.com/cockroachlabs/managed-service/pull/2123

@kannanlakshmi @chrisseto  We need to finalize a naming scheme

Reply:
Sorry Chris, the spreadsheet naming is ad-hoc (the numbers aren't!) so I apologize for that. Can we just say Option 1 , Option 2? I am also okay not calling it anything!

Reply:
@Amruta-Ranade, that's my fault! I got them wrong the first time around. 

I would vote that we stick with small and medium unless there's any major objections?

Reply:
Small and medium seem weird to me -- "Medium" implies there would be a "large" option.
I would reiterate my vote for Option 1/2 :)

Reply:
Since we're bikeshedding, I prefer we don't imply value judgements on size of node. Something that's small for one company could be just fine or medium for another. Option 1/2 avoids that value judgement.